### PR TITLE
docs(blog): add imports

### PR DESCRIPTION
**Pull request by accidentaldeveloper**
Closes: #

- [ ] Docs
- [ ] Tests

### CODE REVIEW

**Comment by accidentaldeveloper** [comment]
Encountered a missing import while following the tutorial. Haven't done full diligence yet but I definitely double checked to make sure it needed to be added.

**Comment by machour** [comment]
Hi, thank you for submission! Are you done with this PR? It's still marked as a draft.

**Comment by accidentaldeveloper** [comment]
> Hi, thank you for submission! Are you done with this PR? It's still marked as a draft.

@machour If you are happy with it, it can be merged but I haven't _thoroughly_ confirmed it's correctness.

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

File: docs/tutorials/blog.md [hunk 1]
```diff
@@ -930,8 +930,12 @@ Let's add some validation before we create the post.
 
 💿 Validate if the form data contains what we need, and return the errors if not
 
-```tsx filename=app/routes/posts/admin/new.tsx lines=[3-9,19-29]
-// ...
+```tsx filename=app/routes/posts/admin/new.tsx lines=[3,8-14,24-34]
+import type { ActionFunction } from "@remix-run/node";
+import { redirect } from "@remix-run/node";
+import { Form, json } from "@remix-run/react";
```
[the remark this answers is not in the document]
Comment (accidentaldeveloper) [reply docs/tutorials/blog.md]: Thank you. The package is still just "remix" in the version I was running for the tutorial, so I made a guess.

**Review by kentcdodds — APPROVED** [review]
Super, thanks!

**Pull request merged**